doc/faq.rst

```diff
@@ -470,6 +470,15 @@ documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-a
 
 .. _faq_mkl_threading:
```

Comment:
Please leave `_faq_mkl_threading` close to "Why does my job use more cores ..."

This is some rst syntax to defined targets and that would mean that links will point to the wrong title